### ISSUE TITLE
Clarify documentation for Task.async_stream/5

### ISSUE THREAD

**Issue by Dalgona** [elixir-lang/elixir#5656]
### Environment

* Elixir & Erlang versions (elixir -v):

    ```
    Erlang/OTP 18 [erts-7.3] [source] [64-bit] [async-threads:10] [hipe] [kernel-poll:false]

    Elixir 1.4.0
    ```

* Operating system:

    Debian GNU/Linux 8.5 (jessie)

### Current behavior

The documentation for `Task.async_stream/5` says:

> [tasks.ex:288:](https://github.com/elixir-lang/elixir/blob/master/lib/elixir/lib/task.ex#L288)
> Each item will be appended to the given `args` and processed by its own task.

And I simply guessed each item in the `enumerable` will be _appended_ so that the item becomes the _last_ item of `args`. But when I actually tried this function, I saw the item had been passed as the first argument to the given `function`, and it makes me a little be confused.

### Expected behavior

I am quite new to functional programming languages, and the word "appending" may have different meaning in this world. But whether it's intended or not, I think this still needs to be clarified by using the opposite word, or by adding some additional explanation.

**Comment by josevalim** [comment]
Oops, the documentation is wrong. Such a small but confusing mistake. I will fix it in master and v1.4, thank you for reporting! :heart:

**Comment by Dalgona** [comment]
Thank you!
